app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolsCategoryDataModel.kt

```diff
@@ -0,0 +1,42 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.map
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import org.ccci.gto.android.common.androidx.lifecycle.combineWith
+import org.ccci.gto.android.common.db.Expression.Companion.constants
+import org.ccci.gto.android.common.db.Query
+import org.ccci.gto.android.common.db.getAsLiveData
+import org.cru.godtools.model.Tool
+import org.keynote.godtools.android.db.Contract
+import org.keynote.godtools.android.db.GodToolsDao
+
+@HiltViewModel
+class ToolsCategoryDataModel @Inject constructor(dao: GodToolsDao) : ViewModel() {
+
+    val selectedCategory = MutableLiveData<String?>(null)
+
+    private val where =
+        Contract.ToolTable.FIELD_TYPE.`in`(*constants(Tool.Type.TRACT, Tool.Type.ARTICLE, Tool.Type.CYOA))
+            .and(Contract.ToolTable.FIELD_HIDDEN.ne(true))
```

Comment:
don't define this as an instance variable, that means every data model that is created will recreate this query, when DB queries are immutable.